COMPATIBILITY.md

```diff
@@ -1,35 +1,36 @@
+# React Native, Expo, and @realm/react compatibility
 
-
-| Realm JavaScript       | React Native       | Expo     | Hermes | npm    | node   |
-|------------------------|--------------------|----------|--------|--------|--------|
-| 12.0.0-alpha.2         | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 12.0.0-alpha.1         | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 11.9.0                 | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 11.8.0                 | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 11.7.0                 | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 11.6.0                 | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 11.5.2                 | >= 0.71.4          | 48       | ✅     | >= 7   | >= 13  |
-| 11.5.1                 | = 0.71.0           | N/A      | ✅     | >= 7   | >= 13 (excluding 19) |
-| 11.5.0                 | = 0.71.0           | N/A      | ✅     | >= 7   | >= 13 (excluding 19) |
-| 11.4.0                 | = 0.71.0           | N/A      | ✅     | >= 7   | >= 13 (excluding 19) |
-| 11.3.1                 | >= 0.70.0          | 47       | ✅     | >= 7   | >= 13 (excluding 19) |
-| 11.2.0                 | >= 0.70.0          | 47       | ✅     | >= 7   | >= 13 (excluding 19) |
-| 11.1.0                 | >= 0.70.0          | 47       | ✅     | >= 7   | >= 13  |
-| 11.0.0                 | >= 0.70.0          | 47       | ✅     | >= 7   | >= 13  |
-| 11.0.0-rc.2            | >= 0.70.0          | N/A      | ✅     | >= 7   | >= 13  |
-| 11.0.0-rc.1            | >= 0.69.0 < 0.70.0 | 46       | ✅     | >= 7   | >= 13  |
-| 11.0.0-rc.0            | >= 0.66.0 < 0.69.0 | 45       | ✅     | >= 7   | >= 13  |
-| 10.24.0                | >= 0.64.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.23.0                | >= 0.64.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.22.0                | >= 0.64.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.21.1                | >= 0.64.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.20.0                | >= 0.64.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.19.5                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.19.4                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.19.3                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.19.2                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.19.1                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 13  |
-| 10.19.0                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 10  |
-| 10.18.0                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 10  |
-| 10.17.0                | >= 0.60.0          | 44       | ❌     | >= 7   | >= 10  |
+| Realm JavaScript    | @realm/react      | React Native       | Expo     | Hermes | npm    | node   |
```

Comment:
Added `@realm/react` but the releases sometimes uses [Realm React](https://github.com/realm/realm-js/releases/tag/realm-react-v0.5.1) and sometimes [@realm/react](https://github.com/realm/realm-js/releases/tag/realm-react-v0.4.2). Which one would you say should be used here?

Reply:
I like it as `@realm/react`